example/spec/kinesisTests/KinesisTestErrorSpec.js

```diff
@@ -119,7 +123,8 @@ describe('The kinesisConsumer receives a bad record.\n', () => {
   });
 
   it('Eventually puts the bad record on the failure queue.', async () => {
-    console.log('\nWait for minimum duration of failure process ~3min');
+    console.log('\nWait for minimum duration of failure process ~3.5 min');
+    await sleep(3.5 * 60 * 1000);
```

Comment:
This is just a fact if we're doing an integration of the 3 lambda failures and retries, it takes at least this long to complete.  I don't want to hide it.